dev/com.ibm.ws.kernel.service/resources/com/ibm/ws/kernel/service/utils/resources/ServiceMessages.nlsprops

```diff
@@ -56,6 +56,18 @@ invalidDuration=CWWKE0107W: The specified parameter does not represent a duratio
 invalidDuration.explanation=The value for the specified configuration parameter is not, or cannot be converted to, a duration value.
 invalidDuration.useraction=Ensure that the value specified in the server configuration is a valid representation of a duration value.
 
+invalidScheduleDelimit=CWWKE0108E: Schedule components must be delimited by a single space ( ). Provided: {0}. Valid example: {1}
+invalidScheduleDelimit.explanation=A schedule must include both a day-of-week and a time component. Optionally, a schedule can also have a timezone component.   
+invalidScheduleDelimit.useraction=Ensure that the schedule specified in the server configuration uses a single space ( ) to delimit day-of-week, time, and timezone components.
+
+invalidScheduleDelimitRange=CWWKE0108E: Schedule ranges must be delimited by a single hyphen (-). Provided: {0}. Valid example: {1}
+invalidScheduleDelimitRange.explanation=A range of days or times must have a start and end component delimited by a single hyphen (-).
+invalidScheduleDelimitRange.useraction=Ensure that the schedule specified in the server configuration uses a single hyphen (-) to delimit day and time ranges.
+
+invalidScheduleParse=CWWKE0109E: Unable to parse the provided parameter to a valid {0}. Provided: {1}. Valid example(s): {2}.
```

Comment:
what are the possible values (or kinds of values) for {0}?

Reply:
They are Java SE class names such as `java.time.DayOfWeek` `java.time.LocalTime` and `java.time.ZoneId`